Dockerfile

```diff
@@ -34,13 +28,17 @@ apt-get install -qyy \
   git \
   curl \
   pre-commit \
-  sudo
+  sudo \
+  gnupg
 apt-get upgrade -qyy \
   rsync
 rm -rf /tmp/* /var/tmp/*
 EOF
 
-RUN apt-get install -y gnupg
+ARG TE_COMMIT=2215fa5c7557b66034068816020f9f611019e457
```

Comment:
Can you add a comment above this line saying where a person should go to pull a new magic commit? I'm guessing we want to track the nemo ci build?